sdk/core/perf/src/program.cpp

```diff
@@ -320,6 +320,13 @@ void Azure::Perf::Program::Run(
       // Take the corresponding proxy from the list in round robin
       parallelTest[i]->SetTestProxy(options.TestProxies[i % options.TestProxies.size()]);
     }
+    if (options.Insecure)
+    {
+#if defined(BUILD_TRANSPORT_WINHTTP_ADAPTER)
+      std::cout << std::endl << " ERROR !!! winHTTP does not support using insecure mode.";
+#endif
+      parallelTest[i]->SetInsecureEnabled(true);
```

Comment:
There are a number of ways a TLS connection can be marked as "insecure". This method should be more specific about what it is doing "DisablePeerCertificateValidation" or "DisableTlsCertificateValidation" or something like that.